playground/infrastructure/verify.py

```diff
@@ -66,7 +72,46 @@ async def _run_and_verify(self, examples: List[Example]):
             examples: beam examples that should be run
         """
 
+
         async def _populate_fields(example: Example):
+            """
+            Populate fields of the example reading them from the backend or from the repository.
+            Args:
+                example: beam example that should be verified
+            """
+            if example.tag.never_run:
+                logging.info("populating example fields from provided files %s", example.filepath)
+                _populate_from_repo(example)
+            else:
+                await _populate_from_runner(example)
+
+        def _populate_from_repo(example: Example):
+            """
+            Populate fields of the example reading them from the repository.
+            Args:
+                example: beam example that should be verified
+            """
+            path = Path(example.filepath)
+            example_folder = path.parent
+
+            log_file_path = example_folder / self.LOGS_FILENAME
+            # Check if the file exists and read its content
+            if log_file_path.exists():
```

Comment:
If these files don't exist, should we be raising some sort of exception?

Reply:
> I'm trying to understand the motivation, could you give me an example of where we would want to use `never_run`?

This is to be able to skip CI and upload logs and output for examples that cannot be run in production. We would like to upload cross-language examples with this tag as we agreed with Kerry.

Reply:
> If these files don't exist, should we be raising some sort of exception?
We do not treat this as exception. It's ok if some files are missing.